scorecard/tests/test_scorecard.py

```diff
@@ -134,3 +135,33 @@ def test_data_explorer_links(self):
         self.assertIn('http://portal:8002/table/cflow/?municipalities=BUF&year=2019&items=3010%2C3030%2C3040%2C3050%2C3060%2C3070%2C3100&amountType=AUDA', element)
         element = selenium.find_elements_by_css_selector(link_class)[19].get_attribute("href")
         self.assertIn('http://portal:8002/table/incexp/?municipalities=BUF&year=2019&items=0200%2C0400%2C1000&amountType=AUDA', element)
+
+    def test_grants_section(self):
+        selenium = self.selenium
+        selenium.get("%s%s" % (self.live_server_url, "/profiles/municipality-BUF-buffalo-city/"))
+
+        element = selenium.find_element_by_css_selector('#income-summary').text
+        self.assertIn('2018-2019 Audited actual', element)
+        self.assertIn('R6 040 712 322', element)
+
+        element = selenium.find_element_by_css_selector('#types-of-transfers').text
+        self.assertIn('2018-2019 Original budget', element)
+        self.assertIn('R183 861 100', element)
+        element = selenium.find_element_by_css_selector('#w-dropdown-list-11').get_attribute("innerHTML")
+        self.assertIn('2018-2019 Original budget', element)
+        self.assertIn('2019-2020 Original budget', element)
+
+        element = selenium.find_element_by_css_selector('#equitable-share').text
+        self.assertIn('2018-2019 Original budget', element)
+        self.assertIn('R79 021 100', element)
+        element = selenium.find_element_by_css_selector('#w-dropdown-list-12').get_attribute("innerHTML")
```

Comment:
this is not a robust selector to use - is there a class you can use to select "the webflow dropdown under #equitable-share"? like `#equitable-share .dropdown`

Reply:
Yes changed to `#equitable-share .dropdown`

Reply:
lol did I guess right?!

Reply:
Yeah I wondered if it was a guess or a polite suggestion